manuscripts/metrics/metrics.py

```diff
@@ -157,12 +157,17 @@ def get_metrics_data(self, query):
         :return: a dict with the results of executing the query
         """
         if self.es_url.startswith("http"):
-            url = self.es_url + '/' + self.es_index + '/_search'
+            url = self.es_url
         else:
-            url = 'http://' + self.es_url + '/' + self.es_index + '/_search'
-        r = requests.post(url, data=query, headers=self.es_headers)
-        r.raise_for_status()
-        return r.json()
+            url = 'http://' + self.es_url
+        es = Elasticsearch(url)
+        s = Search(using=es, index=self.es_index)
+        s = s.update_from_dict(query)
+        try:
+            response = s.execute()
+            return response.to_dict()
+        except Exception as e:
+            raise e
```

Comment:
Capturing the Exception an reraising it without doing any thing is a bit weird. And having the return inside the try also.

Reply:
My bad. I wanted to use the error and show the exact information about the error to the user to make debugging easier.

Reply:
IMHO once this comment is addressed the PR is ready to be merged.